scripts/unzip_backup.py

```diff
@@ -55,13 +55,20 @@ def get_history_file():
 def push_csv_to_db(extracted_csv_path):
     rowcount = 0
     manifest = json.dumps(dict())
+    _, _, files = next(os.walk(extracted_csv_path))
+    if len(files) not in (5, 9):
```

Comment:
what's the significance of 5 and 9? [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)?

Reply:
Uploaded report from koku-metrics-operator always includes 4 or 8 logging tables with 1 manifest file. I add this check to avoid parsing report if there's structure change from koku-metrics-operator.